engine/swarm/services.md

```diff
@@ -288,7 +288,17 @@ $ docker service create \
 Service constraints let you set criteria for a node to meet before the scheduler
 deploys a service to the node. You can apply constraints to the
 service based upon node attributes and metadata or engine metadata. For more
-information on constraints, refer to the `docker service create` [CLI  reference](../reference/commandline/service_create.md).
+information on constraints, refer to the `docker service create` [CLI reference](../reference/commandline/service_create.md).
+
+Placement preferences let you divide tasks evenly over different categories of
+nodes. An example of where this may be useful is balancing tasks between
+multiple datacenters or availability zones. Using a placement preference, you
```

Comment:
Maybe "In this case, you can use a placement preference to spread out tasks to multiple data centers and reduce the risk of a localized outage."

In general, prefer active voice.